yarn.config.cjs

```diff
@@ -179,7 +179,7 @@ async function expectPullRequestTemplate(workspace, workspaceName) {
 
   const pullRequestTemplate = await getWorkspaceFile(
     workspace,
-    '.github/PULL_REQUEST_TEMPLATE.md',
+    '.github/pull_request_template.md',
```

Comment:
Notably, whatever `fs` calls is backing this is newly case-sensitive in Node 22, so this fails if it doesn't actually match the filename.